contracts/src/CrossChainApplications/ERC20Bridge/tests/ERC20BridgeTests.t.sol

```diff
@@ -762,4 +756,10 @@ contract ERC20BridgeTest is Test {
                 )
             );
     }
+
+    function _formatErrorMessage(
```

Comment:
I'm guessing the motivation to put this helper in the tests but not in the contracts themselves is to minimize gas cost?

Reply:
yup exactly